src/components/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  setAvtiveTab: (value: string) => void;
+  avtiveTab: string,
+};
+
+export const Footer: React.FC<Props> = ({ todos, setAvtiveTab, avtiveTab }) => {
+  const tabs: string[] = ['All', 'Active', 'Completed'];
```

Comment:
we can move these fields to some enum and move this constant out of the component